### ISSUE TITLE
Describing distance quantitatively: design question

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#156]
Apologies for long background info!!

@arouinfar, before I started working on Gravity Force Lab Basics (GFLB), I was already thinking that providing quantitative distance values (actual meters from the other mass sphere) would be a very good way to provide distance information in Gravity Force Lab (GFL) regular. Doing so has 2 big benefits:
1. it provides an accurate and decent experience for blind users, and
2. it makes the ruler interaction completely optional for blind users.

**Question**
1. Do you have any concerns pedagogically about providing distance in meters in the described experience of this sim, even though there is no measured distance in meters on the screen like in GFLB? Specifically, GFL regular would sound like GFLB with Distance checkbox checked (that sim's default experience).

**Background to question:** 
In Gravity Force Lab Basics (GFLB), their is an option to show the measured distance (in kilometers) or to hide it. When hidden the arrow measuring and showing the distance between the centers of the mass spheres disappears. Thus, visually, there are two clear modes:
1. a quantitative mode that combines relative distance with the exact kilometers, and 
2. a qualitative mode that just shows the relative distance (with no kilometers shown on the screen).

The Distance checkbox option allowed us to design qualitative distance descriptions that give non-visual users parallel quantitative and qualitative modes while moving the mass spheres or reading through the current state of the sim. 

While  designing distance/position descriptions for GFLB we found that there was no need to provide quantitative position information (the exact kilometer mark). Ultimately, there are a few qualitative position descriptions that provide a sense of where a mass sphere is along the track. These position descriptors are provided whether or not the distance checkbox is checked because they work nicely whether the distance is being described in kilometers or described in relative terms.

**In Gravity Force Lab regular (GFL)...**
There is only the single mode that does not visually provide measured distance on the screen, though you could consider that the ruler provides a natural number line by sitting directly under the mass spheres.

**Note:** There are a couple of edge cases where distance is described qualitatively even in the quantitative mode.

Would love to know about any concerns you have about using GFLB's quantitative distance mode for this sim.

**Comment by terracoda** [comment]
@arouinfar, let me know if you would rather meet synchronously about this issue.
And to get a good feeling about the quantitative distance mode, just try GFLB with distance checked.

**Comment by terracoda** [comment]
@arouinfar, if you need me to simplify this issue, I am really only concerned about the question in https://github.com/phetsims/gravity-force-lab/issues/156#issue-458701273:

> 1. Do you have any concerns pedagogically about providing distance in meters in the described experience of this sim, even though there is no measured distance in meters on the screen like in GFLB? Specifically, GFL regular would sound like GFLB with Distance checkbox checked (that sim's default experience).

Again, apologies for all the background info.

**Comment by terracoda** [comment]
And I should mention that distance is described qualitatively in the screen summary
- m1 and m2 are {{not so close to}} each other, centers exactly {{4 meters apart}}.

**Comment by arouinfar** [comment]
@terracoda sorry for taking a bit of time getting to this, I was out for a conference.

> Again, apologies for all the background info.

No need to apologize, I appreciate the background info!!

> Do you have any concerns pedagogically about providing distance in meters in the described experience of this sim, even though there is no measured distance in meters on the screen like in GFLB? 

My one hesitation is that GFL has a general learning goal around measurement, which is why it uses the ruler instead of the Distance checkbox. By providing the quantitative distance, the need to use the ruler is removed. I wonder if it would be possible/feasible to add the quantitative description of the distance only when the ruler overlaps with the dashed lines extending from the center of mass. 

![image](https://user-images.githubusercontent.com/8419308/60355647-2b27a380-998c-11e9-93ce-1e0adb310f60.png)

**Comment by terracoda** [comment]
> By providing the quantitative distance, the need to use the ruler is removed. 

Yes, that is the intention, to remove the need of using the ruler at all for students who have low enough vision to need to use a screen reader.

@arouinfar, I like your idea above, but I need to know what you want the students to hear when they are moving the spheres, not the ruler. The ruler will involve quantitative meter information being spoken. But what do you want to hear when the spheres are moved: described distance with region names or distance in meters?

If you listen to GFLB with Distance unchecked, you will hear the described distance experience. Would you want that experience to be the default experience in GFL regular? 

Is measured distance a secondary learning goal?

**Comment by terracoda** [comment]
@arouinfar, would you be able to make a design meeting early on a Tuesday morning (8 am mountain time). We could discuss in detail the pros and cons of the ruler interaction and which described experience provides the most useful information when a student is moving the spheres?

**Comment by terracoda** [comment]
**On  Slack I said:**
> I was hoping to just make sure you understood how complicated it might be for a blind student to move the ruler.
When I first started on this sim, Ariel and I think Kathy, indicated that the using the ruler had minimal importance. 

In addition to being minimally important, here are a couple other points: 
- the ruler's placement (just below the spheres) acts as a visual number line that can be used as measurement tool without moving it
- in interviews some students do not realize that they can move it

@arouinfar, would you be able to describe exactly what you would want a student to do with the ruler?
 Then I can consider how the same goals (not necessarily the same interaction/s) can be met by the described experience.

**Comment by terracoda** [comment]
@arouinfar, any thoughts?

**Comment by arouinfar** [comment]
@terracoda I distinctly remember writing up a response to this last week, so apologies for keeping you waiting for so long.

After much thought and your detailed explanations, I think we should provide the quantitative distance descriptions to screen reader users (as is done in GFLB when the Distance checkbox is on).

The learning goals for the ruler are all secondary:
* Determining the functional relationship between force and distance (F ~ 1/r^2)
* Determining the value of the universal gravitational constant, G.
* Setting up controlled experiments (e.g. by incrementally changing distance and leaving other variables constant).

None of these goals are about <i>how</i> to use the ruler. The ruler is merely a means to determining the precise distance between the masses.

Many visual users will likely have a sense of the quantitative distance value without actually using the ruler, given its proximity to the masses and persistent nature. In playing around with the sim, I found that my quick distance estimates were often within in a tick mark or two of the actual value. When trying to be a bit more accurate, I would sometimes just use the cursor to draw a line from the center of mass down to the ruler. It's entirely possible to know the precise distance value without actually moving/using the ruler.

Providing quantitative distance descriptions to visually impaired users would indeed be the most parallel experience to visual users.

**Comment by terracoda** [comment]
@arouinfar, this is great! And thanks so much for your thoughtful response!

@zepumph and I can move forward with the "Distance checked" experience for the spheres, and in this fall, I'll figure out how best the ruler interaction can be described in a way that provides a decent experience (for everyone) as an additional tool. The ruler is already keyboard accessible, I just need to figure out what kinds of things it should be saying as a student moves it around.

**Design Decision:** 
We implement the quantitative distance experience when moving the spheres - the same as in GFLB when the Distance checkbox is checked. The only difference is the units. Here in GFL regular we have meters instead of kilometers.

**Comment by terracoda** [comment]
I am close this issue now.
